testing/marketplace/deployer_helm_tiller_base/onbuild/helm-dependency-build/chart/minimal-helm-chart/templates/pod.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: {{ .Release.Name }}
+  labels:
+    heritage: {{ .Release.Service | quote }}
+    release: {{ .Release.Name | quote }}
+spec:
+  containers:
+  - name: {{ .Release.Name }}
+    image: gcr.io/cloud-marketplace-containers/google/nginx1:latest
```

Comment:
suggestion: use some simple thing like busybox with a sleep script instead of nginx. Just a note that if the script sleeps, make sure it can get interrupted or the pod won't shutdown.

Reply:
Done.